Decred Wallet/Features/Overview/OverviewViewController.swift

```diff
@@ -389,7 +389,10 @@ class OverviewViewController: UIViewController {
     }
     
     @objc func handleBackupWallet() {
-        // TODO: When implementing backup page
+        let backupReminderVC = Storyboards.Backup.instantiateViewController(for: BackupReminderViewController.self).wrapInNavigationcontroller()
```

Comment:
I recommend renaming all instances of `Backup` to `SeedBackup` to prevent any ambiguity.